.idea/codeStyles/Project.xml

```diff
@@ -1,6 +1,22 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
     <JetCodeStyleSettings>
+      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
+        <value>
+          <package name="java.util" alias="false" withSubpackages="false" />
+          <package name="kotlinx.android.synthetic" alias="false" withSubpackages="true" />
+          <package name="io.ktor" alias="false" withSubpackages="true" />
+        </value>
+      </option>
+      <option name="PACKAGES_IMPORT_LAYOUT">
+        <value>
+          <package name="" alias="false" withSubpackages="true" />
+          <package name="java" alias="false" withSubpackages="true" />
+          <package name="javax" alias="false" withSubpackages="true" />
+          <package name="kotlin" alias="false" withSubpackages="true" />
+          <package name="" alias="true" withSubpackages="true" />
+        </value>
+      </option>
```

Comment:
Are these changes really needed?

Reply:
I am not sure, I took a look in the .gitignore file and the code styles are not ignored, in fact they're included. They were created by Intellij, I could take them out.

Reply:
This is added when you import the project for the first time, it is actually correct. When you develop and upgrade the IDE this change should happen, but sometimes it's not triggered until you remove it and reimport the project again. All good here.